test/test_model.py

```diff
@@ -25,7 +30,8 @@ class DummyModel(NubisonModel):
         {"ARTIFACT_DIRS": ",".join(artifact_dirs)}
     ):
         # Register the model
-        register(DummyModel(), model_name=model_name)
+        model_id = register(DummyModel(), model_name=model_name)
+        run_id = get_run_id_from_model_uri(model_id)
```

Comment:
It seems that the **get_run_id_from_model_uri** function directly extracts the run_id from a model_uri formatted as run:/{run_id}/{path}, while also removing the usage of the **get_latest_versions** API, which has been deprecated since MLflow 2.9.0. 

Additionally, I believe that separating production and development environments using tags during model registration and loading could help prevent confusion during team collaboration